tiledb/api/src/array/fragment_info.rs

```diff
@@ -744,18 +744,17 @@ impl Builder {
 #[cfg(test)]
 pub mod tests {
     use super::*;
-    use tempfile::TempDir;
-
     use crate::array::*;
     use crate::config::Config;
     use crate::query::{QueryBuilder, WriteBuilder};
+    use crate::test_util::{self, TestArrayUri};
```

Comment:
Huh, what does `self` mean here?

Reply:
Its the equivalent to `use crate::test_util` so that you can write `test_util::get_uri_generator()`.

Reply:
Neat!